doc/related_projects.rst

```diff
@@ -0,0 +1,29 @@
+================
+Related projects
+================
+
+This is a non-exhaustive list of related, open-source projects.  Each project has strengths/weaknesses
+so it is important to see what project is best for your use case.
+
+
+In the Hyperspy ecosystem
+___________________________________
+- `HyperSpy <https://hyperspy.org>`_: Python library with tools for multi-dimensional data analysis, which pyxem builds upon.
+- `kikuchipy <https://kikuchipy.org/en/stable/>`_: Kikuchi py
```

Comment:
kikuchipy: Python library for processing and analysis of electron backscatter diffraction (EBSD) patterns.